jsonb/src/main/java/io/avaje/jsonb/Jsonb.java

```diff
@@ -107,6 +107,16 @@ static Builder builder() {
     return DefaultBootstrap.builder();
   }
 
+  /**
+   * Get the default Jsonb instance with all generated adapters configured.
+   *
+   * <p>This is a faster alternative to {@code Jsonb.builder().build()} that will return the same
+   * singleton instance.
+   */
+  static Jsonb instance() {
```

Comment:
This should be public right?

Reply:
It's an interface method, which is already public.